plugins/Monolog/Handler/LogCaptureHandler.php

```diff
@@ -21,6 +21,8 @@ class LogCaptureHandler extends AbstractHandler
     public function handle(array $record)
     {
         $this->allLogs[] = $record;
+
+        return false;
```

Comment:
For reference I had used true in the monolog v2 patch: https://sources.debian.org/src/matomo/5.2.2%2Bdfsg-1/debian/patches/0006-Upgrade-monolog-to-v2.patch/#L73